### PR TITLE
PIK-4128: Move internal transfers to separate capability

### PR DESCRIPTION
**Pull request by amirhac**

### CODE REVIEW

File: v2/openapi/fb-unified-openapi.yaml [hunk 1]
```diff
@@ -2890,12 +2895,35 @@ components:
...
+        - $ref: '#/components/schemas/PeerAccountTransferCapability'
+        - type: object
+          required:
+            - accountId
+            - amount
+          properties:
+            accountId:
+              type: string
+            amount:
+              $ref: '#/components/schemas/PositiveAmount'
```
Comment (yanivmo) [inline v2/openapi/fb-unified-openapi.yaml:2922]: Split it into `PeerAccountTransferAddress` that contains only the account ID and `PeerAccountTransferDestination` that inherits `PeerAccountTransferAddress` and adds to it the amount property. This is needed both for the uniformity and for deposit support, that we'll add separately.

File: v2/openapi/fb-unified-openapi.yaml [hunk 2]
```diff
@@ -2905,13 +2933,18 @@ components:
               type: string
             amount:
               $ref: '#/components/schemas/PositiveAmount'
-    CrossAccountTransfer:
+    PeerAccountTransfer:
       type: object
       allOf:
-        - $ref: '#/components/schemas/CrossAccountTransferDestination'
+        - $ref: '#/components/schemas/PeerAccountTransferDestination'
```
Comment (yanivmo) [inline v2/openapi/fb-unified-openapi.yaml:2939]: Add optional reference ID as in fiat transfers.

**Review by yanivmo — CHANGES_REQUESTED** [review]

File: v2/openapi/fb-unified-openapi.yaml [hunk 3]
```diff
@@ -2905,13 +2933,18 @@ components:
               type: string
             amount:
               $ref: '#/components/schemas/PositiveAmount'
```
Comment (yanivmo) [inline v2/openapi/fb-unified-openapi.yaml:2935]: Split it into `InternalTransferAddress` that contains only the account ID and `InternalTransferDestination` that inherits `InternalTransferAddress` and adds to it the amount property.

File: v2/openapi/fb-unified-openapi.yaml (hunk 2 above)
Comment (amirhac) [reply v2/openapi/fb-unified-openapi.yaml:2939]: To the InternalTransfer as well?

File: v2/openapi/fb-unified-openapi.yaml (hunk 1 above)
Comment (amirhac) [reply v2/openapi/fb-unified-openapi.yaml]: Done

File: v2/openapi/fb-unified-openapi.yaml (hunk 3 above)
Comment (amirhac) [reply v2/openapi/fb-unified-openapi.yaml:2949]: Done

File: v2/openapi/fb-unified-openapi.yaml (hunk 2 above)
Comment (amirhac) [reply v2/openapi/fb-unified-openapi.yaml]: Done

**Review by yanivmo — APPROVED** [review]

**Pull request merged**